src/RestApi.ts

```diff
@@ -25,7 +28,9 @@ export class RestApi {
     private readonly _username: string;
     private readonly _basedom: string;
 
-    private _myUserPromise: Promise<any> = null;
+    private _user: any = null;
```

Comment:
Guess we can use `User` type here

Reply:
Not really; we have types for XMPP data objects (User, Call, ...) but not for REST data objects.